src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1132,7 +1132,7 @@ int fromCommandLine (
             try
             {
                 std::stringstream ss;
-                setup = setup_ServerHandler(config, ss);
+                setup = setup_ServerHandler(config, false, ss);
```

Comment:
Put a comment on the `false` param saying what it controls (i.e. `(config, /*acceptPeers*/ false, ss)`).